src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-restore-modal/rbd-trash-restore-modal.component.html

```diff
@@ -0,0 +1,53 @@
+<cd-modal>
+  <ng-container i18n
+                class="modal-title">Restore Image</ng-container>
+
+  <ng-container class="modal-content">
+    <form name="restoreForm"
+          class="form"
+          #formDir="ngForm"
+          [formGroup]="restoreForm"
+          novalidate>
+      <div class="modal-body">
+        <p>
+          <ng-container i18n>To restore</ng-container>
```

Comment:
Please apply the following patch to fix the text display issue.

![auswahl_003](https://user-images.githubusercontent.com/1897962/45826061-f0edef00-bcf3-11e8-87ce-11073064d131.png)

```
--- src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-restore-modal/rbd-trash-restore-modal.component.html	(date 1533458219000)
+++ src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-trash-restore-modal/rbd-trash-restore-modal.component.html	(date 1537454340000)
@@ -10,9 +10,9 @@
           novalidate>
       <div class="modal-body">
         <p>
-          <ng-container i18n>To restore</ng-container>
+          <ng-container i18n>To restore</ng-container>&nbsp;
           <kbd>{{ poolName }}/{{ imageName }}@{{ imageId }}</kbd>,&nbsp;
-          <ng-container i18n>type the image's new name and click</ng-container>
+          <ng-container i18n>type the image's new name and click</ng-container>&nbsp;
           <kbd i18n>Restore Image</kbd>.
         </p>
```